Source/render/NodeRenderer.swift

```diff
@@ -358,3 +362,17 @@ extension NodeRenderer: AnimationObserver {
     }
 
 }
+
+@discardableResult fileprivate func calculateZPosition(_ nodeRenderer: NodeRenderer?, currentIndex: Int = 0) -> Int {
```

Comment:
Are you really going to call this method with null renderer? Does it make any sense?